vagrant/vagrantXterm.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+xterm -e 'vagrant ssh' &
```

Comment:
Why an xterm?  Why not something more general?  xterm isn't available everywhere.  Specifying xterm for OS X is kind of like specifying Apple's Terminal.app for Linux.